addons/binding/org.openhab.binding.freeathome/src/main/java/org/openhab/binding/freeathome/handler/FreeAtHomeBridgeHandler.java

```diff
@@ -337,7 +337,11 @@ private void connectGateway() {
         try {
             // Extract jID
             String jid = this.getJid(m_Configuration.login);
-            m_XmppClient.login(jid, m_Configuration.password);
+            m_XmppClient.login(jid.substring(0, jid.indexOf("@")), m_Configuration.password);
+
+            Presence presence = new Presence(Jid.of("mrha@busch-jaeger.de/rpc"), Presence.Type.SUBSCRIBE, null, null, null, null, Jid.of(jid), null, null, null);
+            m_XmppClient.send(presence);
```

Comment:
Ok. Here I subscribe for a presence @ mrha. Do you know, if presence update is continuously sent?